plugins/GSdx/GSDevice.cpp

```diff
@@ -45,7 +45,7 @@ GSDevice::GSDevice()
 
 GSDevice::~GSDevice()
 {
-	for(auto &t : m_pool) delete t;
+	for(auto t : m_pool) delete t;
```

Comment:
Done, no changes in performance nor code other than adding some `&`, so far so good. 
Everything is managed by the compiler.
Now this opens the question: should I also now implement a non const iterator? It's completely useless IMO and without meta-programming tricks it would consist in a copy paste without `const` keyword of the current one.